pages/common/moe.md

```diff
@@ -0,0 +1,24 @@
+# moe
+
+> A WYSIWYG text editor for ISO-8859-15 encoded text.
+> More information: <https://www.gnu.org/software/moe/moe.html>.
+
+- Edit a file:
```

Comment:
```suggestion
- Edit a file and create a backup ({{insert what the format of the backup is here}} if edited :
```

Obviously tell me what the format is or just change it yourself in a commit.

Reply:
I'd phrase it the same way as in `nano`: `Open nano and create a backup file (file~) when saving edits:`

Reply:
```suggestion
- Open moe and create a backup file (file~) when saving edits:
```